dotnet/src/Extensions/Planning.FlowPlanner/Skills/CheckRepeatStep/config.json

```diff
@@ -0,0 +1,32 @@
+{
+  "schema": 1,
+  "description": "Given the chat history, determine if user would like to execute the previous task for one more time. If not concluded, generate the next message for follow up.",
```

Comment:
Actually there is some overlap between conversation history for ChatSkills and that for ReAct.
But the key is that we manage the scope of chat history so that both ChatCompletion call and ReAct propmt would be concise.